ui-components/customsidebar.md

```diff
@@ -15,6 +15,8 @@ The sidebar API allows developers to add sidebars on editor instances in a simil
 
 This is the syntax for the addSidebar function: `editor.ui.registry.addSidebar(name:String, spec:Object)`
 
+Including the sidebar name in the `toolbar` option will create a sidebar toggle button with the icon specified in the `onSetup` function.
```

Comment:
This isn't quite correct, as the toolbar icon is already created when just registering the sidebar.

Reply:
Maybe instead we could say something like this?
```
When a new sidebar is registered, a corresponding toolbar button is also created using the same name. This button can then be included in the toolbar by adding the name to the [`toolbar`](({{site.baseurl}}/configure/editor-appearance/#toolbar) configuration option.
```

Reply:
Thank you. It is now fixed.